bn/lessons/basics/modules.md

```diff
@@ -82,13 +82,14 @@ iex> %Example.User{}
 iex> %Example.User{name: "Steve"}
 #Example.User<name: "Steve", roles: [], ...>
 
-iex> #Example.User<name: "Steve", roles: [...], ...>
-#Example.User<name: "Steve", roles: [...], ...>
+iex> %Example.User{name: "Steve", roles: [:mananger]}
```

Comment:
Thank you @ogabriel! One small change. I think this should be `manager` (with only 1 `n`).